harvester_e2e_tests/apis/test_images.py

```diff
@@ -71,6 +72,15 @@ def test_create(self, api_client, unique_name, fake_image_file):
             f"Failed to upload fake image with error:{resp.status_code}, {resp.content}"
         )
 
+    def test_create_long_name(self, api_client, unique_name, fake_image_file):
+        resp = api_client.images.create_by_file(
+            "".join(list("".join([unique_name]*6))[0:43]), fake_image_file
```

Comment:
1. Perhaps extract and add a module or class fixture `unique_name_long` and reuse here and in `test_get_and_delete_long_name`?

2. We can define `unique_name_long` fixture like:
   ```
   >>> uname
   'uname-123-456'
   >>> (uname*(43 // len(uname) + 1))[0:43]
   'uname-123-456uname-123-456uname-123-456unam'
   ```
   Thus we don't need to care about `unique_name` length.

Reply:
I've force-pushed and updated a few things :smile: